Fix repo gate protection (#1969)
woodpecker-ci/woodpecker: docs/docs/20-usage/71-project-settings.md

### Protected

> Every pipeline initiated by a user needs to be approved by the owner before being executed.

Author: ```suggestion
Every pipeline initiated by an webhook event needs to be approved, by the project members who have push rights, before being executed.
```

> Every pipeline initiated by an webhook event needs to be approved, by the project members who have push rights, before being executed.

Reviewer: ```suggestion
Every pipeline initiated by an webhook event needs to be approved by a project members with push permissions before being executed.
```

> This can be used if your repository is public to protect the pipeline configuration from running unauthorized changes on third-party pull requests.

Reviewer: ```suggestion
The protected option can be used as an additional review process before running potentially harmful pipelines. Especially if pipelines can be executed by third-parties through pull-requests. 
```
